packages/object-deep-map-values/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = deepMapValues;
+
+function deepMapValues(obj, fn) {
```

Comment:
I know I didn't do this for object-map-keys and other early just utils but most Just utils now check argument types and throw if they are incorrect or missing (for those not using TS)
(example https://github.com/angus-c/just/blob/master/packages/array-mean/index.js#L8-L16)

Reply:
Agreed this should be done.

Added